src/models/machine_learning/InitialConditions.jl

```diff
@@ -0,0 +1,48 @@
+export InitialCondition
```

Comment:
I have a general comment regarding the term "initial condition". Do we refer by this as a list of potential initial conditions (e.g. ice thickness, surface velocities, glacier surface altimetry etc...) that we'd like to match? Or are we just talking about the ice thicknesses? Because if that's the case, I think we should rather call it Initial state.

Reply:
Hmm, it's a bit more nuanced than that. It depends on what observations you have at `t_0`. If you happen to have a DEM, then yes, you automatically know `S` AND your estimated `B`. And in that case, we should have a way to force or nudge the model to match `S` when inverting for `H`. This will give the most  accurate surface gradient and therefore the best `B`. For the vast majority of cases, inverting `H` or `B` is the same thing, because `B` will always be indirectly derived as `S - H`.